docs/GettingStartedDocs/VisualStudioWindows.md

```diff
@@ -0,0 +1,99 @@
+# Using Visual Studio to Develop Enclave Applications for Windows
+
+This walkthrough assumes that you are using Visual Studio on a Windows development
+machine, and want to develop enclaves for Windows.  If you instead want to use Visual
+Studio Code, see the
+[VS Code instructions](https://github.com/openenclave/openenclave/blob/master/devex/vscode-extension/README.md).
+
+## Prerequisites
+
+You will need the following:
+
+- A development machine with support for SGX1 or SGX1 with Flexible Launch Control (FLC) (see [here](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/SGXSupportLevel.md) for how to
+determine this).
+- [Visual Studio Preview](https://visualstudio.microsoft.com/vs/preview/)
```

Comment:
Do we know with which version of VS the required ELF debugging capability will ship?

Reply:
Per discussion, we don't know that yet, so asking developers to install the preview version is the best we can currently do.